nupic/research/temporal_memory.py

```diff
@@ -149,8 +161,10 @@ def computeFn(self,
 
     (_activeCells,
      _winnerCells,
-     predictedColumns) = self.activateCorrectlyPredictiveCells(
+     predictedColumns,
+     orphanCells) = self.activateCorrectlyPredictiveCells(
```

Comment:
Instead of `orphanCells`, I think we should call them `predictedInactiveCells` (more descriptive).